File: FrontandBack/front.cpp
#include "/home/bub/crow/scripts/crow_all.h" 
#include "cam_logic.h" // for function call??

int main()
{
    crow::SimpleApp app;
    CROW_ROUTE(app, "/")([](){
        auto page = crow::mustache::load_text("front.html");
        return page;
    });

    app.port(5000).multithreaded().run();
}

File: FrontandBack/front.py
from flask import Flask, render_template
import subprocess

app = Flask(__name__)

@app.route('/')
def hello():

    return render_template('front.html')

@app.route('stream')
def stream():
    pass

if __name__ == '__main__':
    app.run(host='0.0.0.0', port=5000)   